lib/cmd/fh3/auth.js

```diff
@@ -0,0 +1,43 @@
+/* globals i18n */
+var fhreq = require("../../utils/request");
+var common = require("../../common.js");
+
+module.exports = {
+  'desc' : i18n._('Performs an auth call against an Auth Policy'),
+  'examples' : [{
+    cmd : 'fhc auth --policy=<policy> --app=<app> --device=<device> --params=<params>',
+    desc : i18n._('Performs a auth call against  Auth Policy')}],
+  'demand' : ['policy', 'app', 'device', 'params'],
+  'alias' : {
+    'policy':'p',
+    'app':'a',
+    'device':'d',
+    'params':'p',
+    0 : 'policy',
+    1 : 'app',
+    2 : 'device',
+    3 : 'params'
+  },
+  'describe' : {
+    'policy' : i18n._("Unique 24 character GUID of your Auth Policy."),
+    'app' : i18n._("Unique 24 character GUID of your client application."),
+    'device' : i18n._("Unique 24 character GUID of the device"),
+    'params' : i18n._("The auth parameters specific to the auth policy")
```

Comment:
Can you provide more info here. 
Both device id and params need to be taken from ngui. 
I do not see why someone will want to use this command.

Reply:
What information do you think that we could provide? Do you think we should remove this command?